ios/Classes/FlutterCarplayPlugin.swift

```diff
@@ -501,6 +505,17 @@ public class FlutterCarplayPlugin: NSObject, FlutterPlugin {
 
             result(false)
 
+        case FCPChannelTypes.openExternalMap:
```

Comment:
this opens a url and not a map. The name doesn't match the functionality

Reply:
renamed it to openUrl